File: 프로그래머스/Level1/나누어 떨어지는 숫자 배열.py
# https://programmers.co.kr/learn/courses/30/lessons/12910

def solution(arr, divisor):
    answer = []
    for n in arr:
        if n % divisor == 0:
            answer.append(n)
    if len(answer) == 0:
        return [-1]
    answer.sort()
    return answer

File: 프로그래머스/Level1/나누어 떨어지는 숫자 배열.js
// https://programmers.co.kr/learn/courses/30/lessons/12910

const solution = (arr, divisor) =>
	arr.filter((a) => a % divisor === 0).length === 0
		? [-1]
		: arr.filter((a) => a % divisor === 0).sort((a, b) => a - b);
